sdks/python/setup.py

```diff
@@ -338,6 +338,9 @@ def get_portability_package_data():
               'testcontainers[mysql]>=3.0.3,<4.0.0',
               'cryptography>=41.0.2',
               'hypothesis>5.0.0,<=7.0.0',
+              # Used for integration tests against APIs defined, implemented
+              # and deployed in .test-infra/mock-apis
+              f"mock-apis @ file://{os.path.abspath(os.path.join('..', '..', '.test-infra', 'mock-apis', 'src', 'main', 'python'))}",
```

Comment:
Will this trigger the wheel to install this testing module?

Reply:
Yes, which is why I added it to `extras_require.test` so that in order to install, one has to `pip install ".[test]"`. Being a new thing for me, I tested the difference between `pip install .` and `pip install ".[test]"` in a clean environment.

Using `pip install .` resulted in `Building wheels for collected packages: apache-beam, crcmod, dill, hdfs, pyjsparser, docopt`

Using `pip install ".[test]"` resulted in `Building wheels for collected packages: crcmod, dill, hdfs, apache-beam, mock-apis, pyjsparser, docopt`

Reply:
Ritesh and I did something different in the gen_protos.py.